client/src/components/AddMates/AddMateCard.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  DeleteOutlined,
+  EllipsisOutlined,
+  UserOutlined,
+} from "@ant-design/icons";
+import { Avatar, Card } from "antd";
+import React, { FC } from "react";
+
+import { AddMateType } from "../types";
+
+type AddMateCardProps = {
+  mate: AddMateType;
+  removeMate: (mate: AddMateType) => void;
+};
+
+const { Meta } = Card;
+
+const AddMateCard: FC<AddMateCardProps> = ({ mate, removeMate }) => (
```

Comment:
React.FC is discouraged
https://github.com/facebook/create-react-app/pull/8177

Reply:
sg